index.js

```diff
@@ -0,0 +1,61 @@
+let text = document.getElementById("text");
+let list = document.getElementById("list");
+let search = document.getElementById("search");
+text.value = "";
+search.value = "";
+let index = 0;
+
+text.addEventListener("keypress", function (e) {
+  if (e.key === "Enter") {
+    btnSubmit();
+  }
+});
+
+search.addEventListener("input", function (f) {
```

Comment:
`f` or `e` are not good names for the variable, they don't describe the purpose of the variable. Please give descriptive name.